release/rever.xsh

```diff
@@ -114,7 +114,10 @@ def build_docs():
         make latex
 
         cd _build/latex
-        make
+        # Build with xelatex because pdflatex can not be used with fontspec.
+        # https://github.com/sympy/sympy/issues/20307
+        export LATEXMKOPTS='-xelatex -silent'
```

Comment:
I see this used to be done automatically, but it was changed so that you have to set it in 1f1d754a9d2c0cf5c0bbb899b59a4a9dc4c69108. CC @sylee957 Any reason we can't continue to set it automatically? I actually never understood why we can't just have `make pdf` that runs `make latex` and the latex makefile in one step.

Reply:
The option `-silent` is necessary for documentation release builds because it only fixes the issues with logs for travis tests.
But `-xelatex` option can be bound to the makefile itself. However, I'd be against using string substitutions

Reply:
I think modifying the Makefile is better than having it not work unless you set an environment variable. Although if we had `make pdf` that ran the other Makefile we could just set this variable in the top-level Makefile (not substitutions necessary).

Reply:
Also, it may be possible to set this via a latexmkrc file.

Reply:
I'm open to any suggestions. For now I'm waiting to see if the release script passes with this change.

What I don't understand is why this fontspec issue has come up now.

Reply:
`make latex` generates another Makefile that builds the pdf. Prior to the commit I referenced, the Makefile was automatically modified with sed to use xelatex, but after @sylee957's PR, the sed was removed and you have to set an environment variable manually. 

We really should figure out how to get the release script running on a CI so that we can avoid breaking it. It may also help to somehow share the scripts used on Travis and the release script so that updates to one also change the other.

Reply:
The main manual step involved in running the release script us fixing the authors in .mailmap and AUTHORS.

Sorting that out would mean that every PR from a newer contributor has to update those files.

Reply:
When you run `make latex` it says that `make latexpdf` does this, but that doesn't exist in the Makefile. We should probably update our Makefile to the latest standard sphinx Makefile (i.e., what you'd get from sphinx-quickstart in the latest version of Sphinx). We also include a lot of custom stuff, so this might not be straightforward.